spec/bookshelf-spec.ts

```diff
@@ -1520,11 +1528,11 @@ describe('Bookshelf relations', () => {
   });
 
   it('should give an option to sepcify relation types with a function', () => {
```

Comment:
If my previous comment about `typeForAttribute` affecting both the model type and relation types, these tests should probably verify that. Thoughts?